src/MTGCore.Dtos/MTGCore.Dtos/Models/Cards.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace MTGCore.Dtos
+{
+    public class Cards
+    {
+        public string multiverseid { get; set; }
+        //TODO: make sure automapper takes an integer of ID not string
+        public string id { get; set; }
+        public string name { get; set; }
```

Comment:
Why are the ids being stored as strings?

Reply:
this is because the response from the API is all string therefore the  mapper would only map if they re string. An ideal solution would be to convert the values on retrieval.

Reply:
this will need to be fixed in a future pr

Reply:
I will create an issue